ipn/local.go

```diff
@@ -1022,6 +1011,7 @@ func (b *LocalBackend) authReconfig() {
 	blocked := b.blocked
 	uc := b.prefs
 	nm := b.netMap
+	hasPAC := b.prevIfState.HasPAC()
```

Comment:
`hadPAC`, to match tense

Reply:
I used `hadPAC` in the other chunk for the prior state's HasPAC-edness. In this case, has (present) is correct.